/object_detector:
  ros__parameters:
    debug: true
    hsv_ranges:
      - 31.0
      - 54.0
      - 135.0
      - 255.0
      - 100.0
      - 255.0
